docs/sidebars.js

```diff
@@ -485,115 +485,182 @@ module.exports = {
+                title: "Auth API",
+                slug: "/apis/resources/auth",
+                description:
+                    "The authentication API (aka Auth API) is used for all operations on the currently logged in user. The user id is taken from the sub claim in the token.",
+              },
+              items: require("./docs/apis/resources/auth/sidebar.js"),
+            },
+            {
+              type: "category",
+              label: "Organization Objects",
+              link: {
+                type: "generated-index",
+                title: "Management API",
+                slug: "/apis/resources/mgmt",
+                description:
+                    "The management API is as the name states the interface where systems can mutate IAM objects like, organizations, projects, clients, users and so on if they have the necessary access rights. To identify the current organization you can send a header x-zitadel-orgid or if no header is set, the organization of the authenticated user is set.",
+              },
+              items: require("./docs/apis/resources/mgmt/sidebar.js"),
+            },
+            {
+              type: "category",
+              label: "Instance Objects",
+              link: {
+                type: "generated-index",
+                title: "Admin API",
+                slug: "/apis/resources/admin",
+                description:
+                    "This API is intended to configure and manage one ZITADEL instance itself.",
+              },
+              items: require("./docs/apis/resources/admin/sidebar.js"),
+            },
+            {
+              type: "category",
+              label: "Instance Lifecycle",
+              link: {
+                type: "generated-index",
+                title: "System API",
+                slug: "/apis/resources/system",
+                description:
+                    "This API is intended to manage the different ZITADEL instances within the system.\n" +
+                    "\n" +
+                    "Checkout the guide how to access the ZITADEL System API.",
+              },
+              items: require("./docs/apis/resources/system/sidebar.js"),
+            },
+          ],
         },
         {
           type: "category",
-          label: "OIDC Lifecycle (Beta)",
+          label: "Resource Based (V2)",
+          collapsed: false,
           link: {
             type: "generated-index",
-            title: "OIDC Service API (Beta)",
-            slug: "/apis/resources/oidc_service",
-            description:
-              "Get OIDC Auth Request details and create callback URLs.\n" +
-              "\n" +
-              "This project is in beta state. It can AND will continue breaking until the services provide the same functionality as the current login.",
+            title: "Resource Based APIs (V2)",
+            slug: "/apis/resources/",
+            description: "Resource based API definitions",
```

Comment:
I think we should add a more descriptive description here.
What does resource based API (V2) mean?
When should I use this one?

Maybe we can even link to a more detailed page, where we describe when to use what.